tests/unistd/unlink.c

```diff
@@ -204,10 +199,8 @@ void test() {
 
   err = rmdir("dir-empty");
   assert(!err);
-#ifndef WASMFS
   err = access("dir-empty", F_OK);
   assert(err == -1);
-#endif
```

Comment:
It looks like we only test `F_OK`, at least here. Can we test `R_OK` and friends as well?

Reply:
There is an existing test for that, `test_unistd_access`, which when we get to it will test those things.